src/app/static/src/tests/components/modelCalibrate/modelCalibrate.test.ts

```diff
@@ -83,8 +84,8 @@ describe("Model calibrate component", () => {
         expect(wrapper.find(LoadingSpinner).exists()).toBe(false);
         expect(wrapper.find("#loading-message").exists()).toBe(false);
         const form = wrapper.find(DynamicForm);
-        expect(form.attributes("required-text")).toBe("required");
-        expect(form.attributes("select-text")).toBe("Select...");
+        expect(form.props("requiredText")).toBe("required");
+        expect(form.props("selectText")).toBe("Select...");
```

Comment:
I'm not sure why these switched from attributes to props. I'd like to test both languages but it's stubbornly sticking to English.